edk2toollib/uefi/edk2/parsers/dsc_parser.py

```diff
@@ -92,10 +92,10 @@ def __ParseLine(self, Line, file_name=None, lineno=None):
             if sp is None:
                 raise FileNotFoundError(include_file)
             self.Logger.debug("Opening Include File %s" % sp)
-            self._PushTargetFile(sp)
             lf = open(sp, "r")
```

Comment:
NIT: I know you didn't write this but if we switch to with we can drop lf.close()